nixos/modules/services/web-apps/mastodon.nix

```diff
@@ -892,16 +988,16 @@ in
               ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY="$(cat ${cfg.activeRecordEncryptionDeterministicKeyFile})"
               ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT="$(cat ${cfg.activeRecordEncryptionKeyDerivationSaltFile})"
               ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY="$(cat ${cfg.activeRecordEncryptionPrimaryKeyFile})"
-              SECRET_KEY_BASE="$(cat ${cfg.secretKeyBaseFile})"
+              SECRET_KEY_BASE="$(cat ${cfg.secrets.SECRET_KEY_BASE})"
```

Comment:
```suggestion
```
This is duplicated with the new hunk below.

Reply:
Good catch